docs/designers-developers/developers/block-api/block-deprecation.md

```diff
@@ -194,7 +194,8 @@ E.g: a block wants to migrate a title attribute to a paragraph innerBlock.
 {% ES5 %}
 ```js
 var el = wp.element.createElement,
-	registerBlockType = wp.blocks.registerBlockType;
+	registerBlockType = wp.blocks.registerBlockType,
+	omit = lodash.omit;
```

Comment:
I'm not sure if our documentation should rely on the lodash global variable being available. It would potentially block us from one day WordPress removing lodash. Maybe a better option would be to refactor the sample code to avoid the need for omit function usage?
Any thoughts on this @gziolo, @youknowriad, @aduth?

Reply:
It could be okay to refactor, though I also don't see Lodash going anywhere anytime soon, if ever, as it's offered as a publicly-available script handle in core.

Reply:
We use also lodash in another example. I will land this change as it fixes an issue. If you have some good idea how to improve it further, let’s proceed with follow-up PR.